transform/snowflake-dbt/models/data_quality_layer/analytics/clearbit/dq_on_prem_clearbit.sql

```diff
@@ -0,0 +1,51 @@
+{{config({
+    "materialized": "table",
+    "schema": "data_quality",
+    "tags": "data-quality"
+  })
+}}
+
+
+WITH on_prem_servers AS (
```

Comment:
@ifoukarakis - is there a way to factor out these ("sso_cloud_servers", "on_prem_servers", etc.) into some kind of utils set/library, etc. that we can include so that we can reuse them in multiple scripts. I can see them being used in many other places in the future.

Reply:
IMHO a better approach would be to have the "type" of the server in the `server_fact` table. This would make things simpler.

Another approach would be to move the related parts to a separate model. However I searched for similar occurrences in the project and didn't find something. Let's keep it in mind for now. Perhaps we can create a ticket to revise the `server_fact` table creation.